projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -523,4 +523,21 @@ public function add_settings_page_notice() {
 
 		add_action( 'admin_notices', $admin_notices );
 	}
+
+	/**
+	 * Update Jetpack menu for Atomic.
+	 */
+	public function add_jetpack_menu() {
+		parent::add_jetpack_menu();
+
+		// When the interface is set to wp-admin, we need to link Jetpack stats to Calypso stats.
+		if ( get_option( 'wpcom_admin_interface' ) === 'wp-admin' ) {
+			$this->update_submenus(
+				'jetpack',
+				array(
+					'stats' => 'https://wordpress.com/stats/day/' . $this->domain,
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg)

Should we change the URL to consider another environment like horizon or wpcalypso?

Reply:
I think it's not the case, I saw all wpcom links point just to wpcom. Thanks for the suggestion